src/librbd/CMakeLists.txt

```diff
@@ -188,12 +188,12 @@ endif()
 if(WITH_EVENTTRACE)
   add_dependencies(rbd_internal eventtrace_tp)
 endif()
-target_link_libraries(rbd_internal PRIVATE
+target_link_libraries(rbd_internal
```

Comment:
i don't quite understand this change.

Reply:
@tchaikov if we remove PRIVATE in line 196, we need to remove the PRIVATE here. They need to keep alignment. (Both PRIVATE or Both nothing). Otherwise, it will have compile issue.